ansible/roles/ovos_installer/templates/virtualenv/core-requirements.txt.j2

```diff
@@ -4,6 +4,7 @@ ovos-dinkum-listener[extras,linux]
 ovos-docs-viewer
 ovos-PHAL[linux]
 ovos-phal-plugin-ipgeo
+ovos-tts-plugin-piper
```

Comment:
should be skipped if lang is catalan galician or basque since piper doesnt support those  *

* catalan is supported but pronounced everything wrong